charts/vault/templates/vault-rb.yaml

```diff
@@ -0,0 +1,16 @@
+{{- range $ns := .Values.plugins.vaultPluginSecretsKubernetesReader.namespaces }}
```

Comment:
Maybe only if cluster flag is false?

Reply:
either way is fine... no harm in having cluster-role and per namespace role I think... but as you prefer

Reply:
It might cause confusion, but good enough for now.